Formula/himalaya.rb

```diff
@@ -18,7 +18,7 @@ class Himalaya < Formula
 
   on_linux do
     depends_on "pkg-config" => :build
-    depends_on "openssl@1.1"
+    depends_on "openssl@3" => :dev
```

Comment:
What does `:dev` mean?

Reply:
…means nothing, looks like I've got been distracted and wrote this while I wanted to put `:build`. It's strange though that it caused no issue with the CI, right?